src/20240602_AaveV3Ethereum_GhoStewardUpdate/GhoStewardUpdate.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Gho Steward Update"
+author: "ACI"
+discussions: "https://governance.aave.com/t/arfc-gho-stewards/16466/11"
+---
+
+## Simple Summary
+
+This proposal seeks to update the GHO steward to an upgraded version, allowing more granular control over GHO to increase it’s efficiency
```

Comment:
I would not use the term upgrade, as it could be confused with upgrading the proxy and in the case of gho steward its not a proxy.

Reply:
solved